### ISSUE TITLE
Zulip isn't internationalised

### ISSUE THREAD

**Issue by kostich** [zulip/zulip-desktop#2]
It doesn't seem that the Zulip is internationalised and ready for localisation.

It should be internationalised so it can become best open source solution for Skype-like enterprise communication.

**Comment by timabbott** [comment]
Definitely an issue; I think most of the work overall to internationalize Zulip will be in the server+webapp, though, not the desktop app wrapper (though there is a bit there).

**Comment by lfranchi** [comment]
Qt has built-in support for translations. The work here is basically around:

1. Wrapping all user-visible strings in `tr()`
2. Doing the work to package localization files during deployment
3. Setting up something like https://www.transifex.com/ to allow users to crowd-source translations and periodically include them in our build / distributions

**Comment by kostich** [comment]
As a translator on Transifex, I would like to say that the translation process should be done there. Transifex is a very comfy and fast way for translating.

**Comment by lfranchi** [comment]
Agreed, Transifex would make sense, both for the desktop app and for the zulip web app as well.

**Comment by timabbott** [comment]
Adding a pointer to https://github.com/zulip/zulip/issues/132 since progress on i18n for Zulip is being tracked there.
